### DISCUSSION TITLE
Does Vaultwarden have the Bitwarden team features?

### DISCUSSION THREAD

**Discussion by chrisdietr** [dani-garcia/vaultwarden#1846] [Q&A] [answered]
Hello,
I'm looking for a PW management solution for on online community of several hundred members where every member can potentially have their own account for password management.
There is no revenue attached to the project but I like to follow some best practices when it comes to password management.
Does vaultwarden provide the team features for free (creating groups/teams, sharing passwords in groups)? I know I'd have to pay if I'd use the self hosted Bitwarden/Passbolt version.

Thanks,
Chris

**Comment by jjlin** [comment, marked as answer]
Yes, the supported features are at https://github.com/dani-garcia/vaultwarden/wiki#supported-features.

**Reply by chrisdietr** [reply]
Thank you very much. That looks great - even the send feature is supported.
